python/openvino_tokenizers/cli.py

```diff
@@ -58,24 +58,54 @@ def get_parser() -> ArgumentParser:
             "Example: `convert_tokenizer SimianLuo/LCM_Dreamshaper_v7 --subfolder tokenizer`"
         ),
     )
-    parser.add_argument(
+    special_tokens_group = parser.add_mutually_exclusive_group()
+    special_tokens_group.add_argument(
+        "--add-special-tokens",
+        "--add_special_tokens",
+        required=False,
+        action="store_true",
+        help=(
+            "Tokenizer will add special tokens during tokenization, similar to "
+            "huggingface_tokenizer.encode(texts, add_special_tokens=True). Not affects tiktoken-base tokenizers. "
+            "Not compatible with --not-add-special-tokens."
+        ),
+    )
+    special_tokens_group.add_argument(
         "--not-add-special-tokens",
         "--not_add_special_tokens",
         required=False,
         action="store_false",
+        default=False,
         help=(
             "Tokenizer won't add special tokens during tokenization, similar to "
-            "huggingface_tokenizer.encode(texts, add_special_tokens=False). Not affects tiktoken-base tokenizers."
+            "huggingface_tokenizer.encode(texts, add_special_tokens=False). Not affects tiktoken-base tokenizers. "
+            "This is the default behaviour, so adding this option has no effect, for backward compatibility only. "
```

Comment:
Looks like it's not aligned with HF
![image](https://github.com/openvinotoolkit/openvino_tokenizers/assets/2566854/b7e09fc7-6761-4862-b5d9-9bb1a0259979)
why do we need to skip special tokens during tokenization? We have talked only about detokenization..

Reply:
You wanted to add them manually as I remember. Too many variants are discussed without a clear summary: what do we want.